### ISSUE TITLE
Fio tests gets stuck while communicating through RORA.

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/alba#327]
When performing the fio read test to see communication via RORA we see this error:
```
2016-08-12 15:19:27 014250 +0200 - cmp02 - 31130/0x00007f7379ffb700 - volumedriverfs/AlbaProxyClient - 0000000000014058 -  info - logMessage: RDMAProxy_client(172.19.12.141, 26204)
2016-08-12 15:19:27 014511 +0200 - cmp02 - 31130/0x00007f7379ffb700 - volumedriverfs/AlbaProxyClient - 0000000000014059 -  info - logMessage: connecting
2016-08-12 15:19:27 019315 +0200 - cmp02 - 31130/0x00007f7379ffb700 - volumedriverfs/AlbaProxyClient - 000000000001405a -  info - logMessage: RoraProxy_client(...)
2016-08-12 15:19:27 019401 +0200 - cmp02 - 31130/0x00007f7379ffb700 - volumedriverfs/BackendInterface - 000000000001405b -  warning - do_wrap_: Retrying with new connection (retry: 1, sleep before retry: 1000 milliseconds)
2016-08-12 15:19:28 020282 +0200 - cmp02 - 31130/0x00007f734cff9700 - volumedriverfs/(UnspecifiedComponent) - 000000000001405c -  error - 
 Alert !!imsg has error in retval

2016-08-12 15:19:28 020439 +0200 - cmp02 - 31130/0x00007f7379ffb700 - volumedriverfs/(UnspecifiedComponent) - 000000000001405d -  error - giocbp->request_->_failed is true. Error is -5
2016-08-12 15:19:28 020518 +0200 - cmp02 - 31130/0x00007f7379ffb700 - volumedriverfs/AlbaProxyClient - 000000000001405e -  info - logMessage: retcode:-5
2016-08-12 15:19:28 022090 +0200 - cmp02 - 31130/0x00007f734cff9700 - volumedriverfs/(UnspecifiedComponent) - 000000000001405f -  error - 
 Alert !!imsg has error in retval

2016-08-12 15:19:28 022233 +0200 - cmp02 - 31130/0x00007f7337fff700 - volumedriverfs/(UnspecifiedComponent) - 0000000000014060 -  error - giocbp->request_->_failed is true. Error is -5
2016-08-12 15:19:28 022322 +0200 - cmp02 - 31130/0x00007f7337fff700 - volumedriverfs/AlbaProxyClient - 0000000000014061 -  info - logMessage: retcode:-5
2016-08-12 15:19:28 023661 +0200 - cmp02 - 31130/0x00007f734cff9700 - volumedriverfs/(UnspecifiedComponent) - 0000000000014062 -  error - 
 Alert !!imsg has error in retval
```

The error seems to be connected to errors on the ASDS:
```
/var/log/upstart/alba-asd-cPlf0CcffMlb6EHfwjRhJMgC63I50qp3.log:[2016-08-12 15:00:13.095500] [0x000007f97d1ffb70] [error]   IOExecFileRead failed with error -5
/var/log/upstart/alba-asd-cPlf0CcffMlb6EHfwjRhJMgC63I50qp3.log:[2016-08-12 15:00:13.096712] [0x000007f97d2ffd70] [error]   failed to open file=/mnt/alba-asd/lnXKdMyvJPu13gJL/cPlf0CcffMlb6EHfwjRhJMgC63I50qp3/blobs/00/00/00/00/00/00/47/0000000000000036 flags=0 mode=436 errno=2
/var/log/upstart/alba-asd-cPlf0CcffMlb6EHfwjRhJMgC63I50qp3.log:[2016-08-12 15:00:13.096733] [0x000007f97d2ffd70] [error]   IOExecFileRead failed with error -5
/var/log/upstart/alba-asd-cPlf0CcffMlb6EHfwjRhJMgC63I50qp3.log:[2016-08-12 15:00:13.097977] [0x000007f97d37fe70] [error]   failed to open file=/mnt/alba-asd/lnXKdMyvJPu13gJL/cPlf0CcffMlb6EHfwjRhJMgC63I50qp3/blobs/00/00/00/00/00/00/47/0000000000000036 flags=0 mode=436 errno=2
/var/log/upstart/alba-asd-cPlf0CcffMlb6EHfwjRhJMgC63I50qp3.log:[2016-08-12 15:00:13.097994] [0x000007f97d37fe70] [error]   IOExecFileRead failed with error -5
/var/log/upstart/alba-asd-cPlf0CcffMlb6EHfwjRhJMgC63I50qp3.log:[2016-08-12 15:00:13.099582] [0x000007f97d97ca70] [error]   failed to open file=/mnt/alba-asd/lnXKdMyvJPu13gJL/cPlf0CcffMlb6EHfwjRhJMgC63I50qp3/blobs/00/00/00/00/00/00/47/0000000000000036 flags=0 mode=436 errno=2
/var/log/upstart/alba-asd-cPlf0CcffMlb6EHfwjRhJMgC63I50qp3.log:[2016-08-12 15:00:13.099605] [0x000007f97d97ca70] [error]   IOExecFileRead failed with error -5

```

An also unhappy result was that the read errors resulted in the fio test being totally stuck. It seems like the manifest cache in the proxy is not updated after such errors.

**Comment by domsj** [comment]
workaround: don't use rora!
(glad I could help)

**Comment by kinvaris** [comment]
Should be fixed in combination of `alba 0.9.18` and new volumedriver updates referenced in https://github.com/ovs-projects/org_quality/issues/22

**Comment by toolslive** [comment]
wrong path calculation is the cause of this.

**Comment by toolslive** [comment]
fix in #344

**Comment by kinvaris** [comment]
Not seen anymore during test with alba version:
```
root@str03:~# alba version
-1.-1.-1
git_revision: "remotes/origin/master-0-g2f79834"
git_repo: "https://github.com/openvstorage/alba.git"
compile_time: "30/08/2016 09:03:31 UTC"
```
